tools/webpack-plugin/src/models/AssetStats.ts

```diff
@@ -0,0 +1,9 @@
+import { UploadResult } from '@backtrace/sourcemap-tools';
+
+export interface AssetStats {
+    readonly debugId: string;
+    sourceSnippet?: boolean | string | Error;
```

Comment:
1. why we should continue generating source maps information if we receive an error? 
2. why are we making this object so hard to maintain. Why do we want to print logs in the end instead of showing them when they happened? It's not natural. We can show the summary however a list of errors in the end might be huge if sourcemap cannot work for some kind of reason (let's say permission denied)